lib/bootloader_setup.pm

```diff
@@ -281,12 +281,14 @@ sub bootmenu_type_extra_boot_params {
 }
 
 sub bootmenu_type_console_params {
+    my ($baud_rate) = shift // '';
+    $baud_rate = $baud_rate ? ",$baud_rate" : '';
```

Comment:
I suspect that 'baud' is some kinda of abbreviation, right? But I can only guest what it could mean. Can we come up with more meaningful name for variable? I could suggest something when you will decode for me what actually baud_rate suppose to mean :)

Reply:
It's actually not, and that's the exact name of parameter which you pass after serial device: https://en.wikipedia.org/wiki/Baud

Reply:
was not aware of that. Maybe you need to add wiki link to our code? ( joking :) )